.github/actions/deploy-macosx/action.yml

```diff
@@ -10,7 +10,9 @@ runs:
         export MACOSX_DEPLOYMENT_TARGET=10.13
         echo "MACOSX_DEPLOYMENT_TARGET=$MACOSX_DEPLOYMENT_TARGET" >> $GITHUB_ENV
 
-        brew install boost ccache gcc swig autoconf-archive automake cmake libomp libtool libusb ant maven nasm xz pkg-config sdl gpg1 bison flex perl ragel binutils gradle gmp isl libmpc mpfr
+        brew install boost ccache gcc swig autoconf-archive automake cmake libomp libtool libusb ant maven nasm xz pkg-config sdl gpg1 bison flex perl ragel binutils gradle gmp isl libmpc mpfr zlib curl pcre libffi yajl ssdeep luarocks
+        brew install geoip --with-geoipupdate
+        brew install doxygen --with-llvm
```

Comment:
Why is the set of dependencies different on Linux and Mac? It looks like it's building without any of those dependencies too. Which ones do you think we actually need?

Reply:
If you are talking about geoip and oxygen they are installing on Linux too. For Mac, they have separate lines just because they need additional args for installation. Or you meant something else?

Reply:
Well, it doesn't look like Lua or libffi gets used for anything, while curl and zlib come with the system, so I'll probably just leave those out. I also don't think we need to geoipupdate or LLVM in there either. Sounds good?

Reply:
I'm also not sure we need doxygen to build an installation of the project, see https://github.com/SpiderLabs/ModSecurity#Dependencies

Doxygen is used to create api documentation from the source code as well as other documentation pages. Shouldn't be necessary to build modsecurity.

Reply:
Sorry for the late response. @saudet yes it sounds good. I see that you already fixed it. Thanks! Once again sorry for the late response, I was AFK last week.